### DISCUSSION TITLE
how to properly make a 404 page for seo

### DISCUSSION THREAD

**Discussion by kzyq** [quasarframework/quasar#15556] [CLI - SSR mode]
Hello,
all methods proposed in the Quasar documentation return response status 200 OK
How to return status 404 without changing URL?

For example
We have the URL http://example.com/productname/smartwatch
we find the product and display the page,
but when the user enters the page
http://example.com/productname/smartwath
we should return response code 404 so that the search bot knows that the given page does not exist and does not index it by google bot

example

![Zrzut ekranu 2023-03-14 102957](https://user-images.githubusercontent.com/46751932/224957773-dc7e8302-7c2a-4462-8f14-9002b3beff19.jpg)

**Comment by hillerabde76** [comment]
Creating a proper 404 page is important for both user experience and SEO. When a user lands on a 404 page, it means that the page they are trying to access cannot be found. Here's how to create a 404 page that is both helpful for users and SEO-friendly, using "[security guard service Los Angeles](https://prosecurityguard.com/)" as an example:

Make it clear that the page cannot be found: Your 404 page should clearly communicate to the user that the page they are looking for cannot be found. You can use a message like "Oops! We couldn't find the page you're looking for" or "Sorry, the page you requested cannot be found."

Provide options for the user: Instead of leaving the user stranded on a dead-end page, provide them with options to navigate to other pages on your website. You can include links to your homepage, services page, contact page, or any other relevant pages.

Use your target keywords: You can use your target keywords, such as "security guard service Los Angeles", in the copy on your 404 page. This can help with SEO by indicating to search engines that your website is relevant to those keywords.

Use your branding: Your 404 page should also include your branding elements, such as your logo and color scheme. This can help reinforce your brand identity and make the user feel more connected to your website.

Monitor your 404 errors: Finally, it's important to regularly monitor your website for 404 errors and fix them as soon as possible. This can help prevent users from getting frustrated and leaving your website, which can negatively impact your SEO and overall user experience.

Overall, creating a helpful and SEO-friendly 404 page can help improve your website's user experience and search engine visibility. By incorporating your target keywords and branding, you can also reinforce your website's relevance and authority in your industry, such as "security guard service Los Angeles".
